diff --git a/node_modules/expo-video-audio-extractor/android/src/main/java/expo/modules/videoaudioextractor/ExpoVideoAudioExtractorModule.kt b/node_modules/expo-video-audio-extractor/android/src/main/java/expo/modules/videoaudioextractor/ExpoVideoAudioExtractorModule.kt
index ccc9ae0..5a4f265 100644
--- a/node_modules/expo-video-audio-extractor/android/src/main/java/expo/modules/videoaudioextractor/ExpoVideoAudioExtractorModule.kt
+++ b/node_modules/expo-video-audio-extractor/android/src/main/java/expo/modules/videoaudioextractor/ExpoVideoAudioExtractorModule.kt
@@ -35,7 +35,7 @@ class ExpoVideoAudioExtractorModule : Module() {
           promise.resolve(p.output)
         } catch (t: Throwable) {
           sendEvent("error", mapOf("taskId" to taskId, "message" to t.message))
-          promise.reject("EXTRACT_ERROR", t)
+          promise.reject("EXTRACTION_ERROR", t.message, t)
         } finally {
           jobs.remove(taskId)
         }
